g++  -I /usr/local/lib/python2.7/dist-packages/ms_ivy-1.7.0-py2.7.egg/ivy/include -L /usr/local/lib/python2.7/dist-packages/ms_ivy-1.7.0-py2.7.egg/ivy/lib -Wl,-rpath=/usr/local/lib/python2.7/dist-packages/ms_ivy-1.7.0-py2.7.egg/ivy/lib -g -o test_v2 test_v2.cpp -lz3 -pthread
test_v2.ivy: line 359: error: assertion failed
g++  -I /usr/local/lib/python2.7/dist-packages/ms_ivy-1.7.0-py2.7.egg/ivy/include -L /usr/local/lib/python2.7/dist-packages/ms_ivy-1.7.0-py2.7.egg/ivy/lib -Wl,-rpath=/usr/local/lib/python2.7/dist-packages/ms_ivy-1.7.0-py2.7.egg/ivy/lib -g -o test_v3 test_v3.cpp -lz3 -pthread
PRISM_DIR=/usr/local/prism-src/prism PRISM_MAINCLASS=simulate.GetProbability bin/run
bin/run: 21: [[: not found
New highest probability: 1.133191e-66 at line 1
New highest probability: 2.157799e-66 at line 32
New highest probability: 5.031051e-66 at line 51
New highest probability: 3.623868e-65 at line 66
New highest probability: 1.079714e-64 at line 103
Total probability: 6.879804e-64
starting to run initial test
finished initial test
The iters recorded for this initial example is 196
starting to run rest of tests
./test_v3 iters=245.0 seed=3131 runs=870
finished randomized testing

The traces recorded and the information on those traces are stored in 'trace_list.txt'

The traces by themselves are found in 'reaction_list.txt'
